### ISSUE TITLE
Cannot login - endless loading

### ISSUE THREAD

**Issue by nichthinterfragen** [furlongm/patchman#556]
Hello,

I have installed Patchman server & client on a Ubuntu Server 22.04 LTS VM following the guide in the installation documentation for Ubuntu 22.04. I didn't yet change the database from sqlite. I'm just trying out Patchman (so no productive installation).

After installation I created a superuser. I'm able to open the web page of Patchman however I cannot login. After entering the right login details of my superuser it just keeps loading without ever timing out. No error message either. 

When I enter the wrong login details the Patchman web interface is quick to load and tell me that the login details are wrong.

Where can I find logs? I don't see anything in /etc/patchman, /var/lib/patchman or /usr/share/patchman nor in /var/log. Apache Logs are not helpful. Error log is empty or contains errors unrelated. Access log only shows the normal access to the web page. 

Any way to debug this?

**Comment by furlongm** [comment]
Can you add `DEBUG = True` in `/etc/patchman/local_settings.py`? That should help debug

**Comment by nichthinterfragen** [comment]
I've changed DEBUG = False to DEBUG = true in the config file. I can login now and it is swift to load. Very weird. Is there a file or something where the debug information is stored? Looking at the web interface itself there is to be no debug output on it.

**Comment by furlongm** [comment]
Anything in `/var/log/apache2/*.log`?

**Comment by nichthinterfragen** [comment]
Sorry for the late reply. I moved on to monitoring the Linux update status using our already existing Zabbix instance.

I check the Apache logs. Nothing really strange. A few 404 because of missing favicon. Some 302. Rest is all 200. Error log contains nothing that is related to this issue(s).

I just installed the latest updates for the OS and rebooted. Things that were buggy before seem to work now. Like creating a OS group and adding an OS to it. Now a few other things are buggy still. However they seem to work after some time and a few reloads.

I'm really not sure what is going on here. Thank you for helping out. For now I'll stick to using Zabbix as it is already integrated on our Linux servers.
